docs/UserGuide/Monitor-Alert/Apache-IoTDB-ConfigNode-Dashboard.json

```diff
@@ -1190,11 +1190,11 @@
             "uid": "${DS_PROMETHEUS}"
           },
           "editorMode": "code",
-          "expr": "sum(rate(process_cpu_time{instance=\"$instance\",name=\"process\"}[1m]))*60",
```

Comment:
for cpu time, FYI

https://stackoverflow.com/questions/58998202/does-multithreading-increase-cpu-time

so it cannot be divided by the number of cpu cores

Reply:
I see. It has been withdrawn.